src/screens/searchResult/screen/tabs/communities/container/communitiesResultsContainer.js

```diff
@@ -0,0 +1,144 @@
+import { useState, useEffect } from 'react';
+import { withNavigation } from 'react-navigation';
+import { useSelector, useDispatch } from 'react-redux';
+import { useIntl } from 'react-intl';
+
+import ROUTES from '../../../../../../constants/routeNames';
+
+import { getCommunities, getSubscriptions } from '../../../../../../providers/hive/dhive';
+
+import {
+  subscribeCommunity,
+  leaveCommunity,
+} from '../../../../../../redux/actions/communitiesAction';
+
+// const DEFAULT_COMMUNITIES = [
```

Comment:
shouldn't this be uncommented?

Reply:
We use top 20 ranked communities for the initial state because of the promise bug.

Reply:
hmm in that case, please randomize result.